content/en/docs/setup/upgrade/index.md

```diff
@@ -1,10 +1,67 @@
 ---
```

Comment:
netlify is complaining about the file name: ```7:07:35 AM: WARN 2020/04/10 14:07:35 Content directory "/opt/build/repo/content/en/docs/setup/upgrade" have both index.* and _index.* files, pick one.
```

Reply:
That was present prior to this, although I can fix it in another pr

Reply:
not needed thanks.